pkg/cvo/sync_worker.go

```diff
@@ -505,8 +515,14 @@ func (w *SyncWorker) syncOnce(ctx context.Context, work *SyncWork, maxWorkers in
 			return err
 		}
 
+		profile, err := w.clusterProfileRetriever.RetrieveClusterProfile(ctx)
+		if err != nil {
+			klog.Warningf("failed to retrieve profile: %v", err)
```

Comment:
I don't think we should `klog` here and then return the `err`.  The `syncOnce` caller is in charge of logging and otherwise announcing the error, as it does [here][1].  Can we drop this line?

[1]: https://github.com/openshift/cluster-version-operator/blob/a49fef5c66c6b0707c54fd93f84d2f51d3d28aca/pkg/cvo/sync_worker.go#L336

Reply:
done :+1: